src/Presentation/TwigEnvironmentConfigurator.php

```diff
@@ -61,6 +61,7 @@ public function getEnvironment( Twig_Environment $twig, array $loaders, array $e
 
 	private function setDefaultTwigVariables( Twig_Environment $twig ): void {
 		$twig->addGlobal( 'basepath', $this->config['web-basepath'] );
+		$twig->addGlobal( 'appconfig', $this->config );
```

Comment:
You missed [one level of inflection](https://github.com/wmde/FundraisingFrontend/pull/909/files#diff-b27bd68e2f2f9595828e438d3c9d2e24R300) - but sure, this is clearly up for a change, threw it in here for prototypical purposes.
I think
* we may have to look a the (payment) config anyways as there are differences of applicability of payment types for the different checkouts (donation, memberships) which are currently not reflected therein. Wanted to discuss that first before elaborating (potentially in the wrong direction)
* list of applicable payment types should definitely be figured out / injected into the view in/depending on the use case, not globally